security/permissions.rst

```diff
@@ -0,0 +1,103 @@
+
+Permissions (AccessPolicy)
+--------------------------
+Firely Server supports filtering the requested client scopes by using built-in custom access policy resources. Both Smart-V1 and Smart-V2 scopes are supported. 
+The resource definitions are preloaded in Firely Server and can be viewed on the administration endpoint API or in Simplier under 
+`Firely Server Definitions - Access Policy <https://simplifier.net/Vonk-ResourcesR4/~resources?text=access&fhirVersion=R4&sortBy=RankScore_desc>`_ .
```

Comment:
When checking the resource on Simplifier, I see a warning for the AccessPolicyDefinition : 
`The canonical from this resource does not match any claim in this context and conflicts with a claim from another scope.

http://hl7.org/fhir

Canonical claims are used to verify ownership of your canonical URLs.`
When I check the URL field I see indeed:
`"url": "http://hl7.org/fhir/StructureDefinition/AccessPolicyDefinition",`
Also, when checking the "AccessPolicy", I see that the `AccessPolicy.instantiatesCanonical` field refers to a target profile of type: `"http://fire.ly/fhir/StructureDefinition/AccessPolicyDefinition"`

I am not that familiar with FHIR, but it does not seem normal.
Could you confirm?

Reply:
@alexzautke. I am not familiar with Simplifier enough to evaluate this. Do you know?